schema-derivatives-perpfutures.graphql

```diff
@@ -1797,6 +1797,9 @@ type PositionSnapshot @entity(immutable: true) {
   " Notional value of collateral position, in USD "
   collateralBalanceUSD: BigDecimal!
 
+  " Realised profit and loss in USD at the time of close or event "
+  realisedPnlUSD: BigInt
```

Comment:
Suggest to change realisedPnlUSD type from BigInt to BigDecimal to be consistent with other USD's type.

Reply:
done